frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -4647,7 +4647,7 @@ export const widgets = [
     styles: {
       dividerColor: {
         type: 'color',
-        displayName: 'Divider Color',
+        displayName: 'Divider color',
```

Comment:
Hey @deanurag, please revert this change.

Reply:
I have reverted the asked change.

Reply:
I have reverted the asked change.